tools/fixup-wheels.sh

```diff
@@ -12,6 +12,7 @@ set -e
 wheelhouse=$1
 fixed_wheelhouse=$2
 platform=$(uname -s)
+gen_record_row=$(dirname $(readlink -f $0))/generate-record-row.py
```

Comment:
nit: readlink is a gnu extension and is not generally available. But since we're only doing this on linux and osx I guess we're fine?

Reply:
suggested alternative?

Reply:
Since we know the script is called from the top-level dir we can just do $PWD/tools/generate-record-row.py